source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
This whole thing is pretty gross...

This looks like a problem.
https://github.com/mixpanel/mixpanel-js/blob/master/build/mixpanel.cjs.js#L5478